protovalidate/internal/rules.py

```diff
@@ -1094,8 +1082,8 @@ def __init__(
     def validate(self, ctx: RuleContext, message: message.Message):
         if not message.HasField(self._field.name):
             return
-        rules = self._factory.get(self._field.message_type)
-        if rules is None:
+        rules: list[Rules] = self._factory.get(self._field.message_type)
+        if not rules:
```

Comment:
noticed we weren't exiting early here, since `self._factory.get` is never returning `None`. Switched to check if we have any rules in the list.